.github/workflows/ci.yml

```diff
@@ -128,6 +128,27 @@ jobs:
       - run: python -m pytest modin/experimental/pandas/test/test_io_exp.py
         if: matrix.part == 3
       - run: bash <(curl -s https://codecov.io/bash)
+  test-experimental:
+    needs: [lint-commit, lint-flake8, lint-black, test-api, test-headers]
+    runs-on: ubuntu-latest
+    env:
+      MODIN_ENGINE: "python"
+      MODIN_EXPERIMENTAL: "True"
+      MODIN_MEMORY: 1000000000
+    name: test experimental
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+      - uses: actions/setup-python@v1
+        with:
+          python-version: "3.7.x"
+          architecture: "x64"
+      - run: pip install -r requirements.txt
+      - run: python -m pytest modin/pandas/test/test_dataframe.py::TestDataFrameMapMetadata
```

Comment:
Why are you test only `TestDataFrameMapMetadata`?

Reply:
This is a small subset, originally proposed by Devin.
I don't want to run the whole testing on experimental, as this would take too much time.

The tests here should be considered as kind of "smoke testing" meaning that experimental mode is not _completely_ broken, not as tests showing that it works - that's different amount of testing.